OpenCL_C.txt

```diff
@@ -3180,6 +3180,20 @@ following forms whose meanings are described elsewhere:
 #pragma OPENCL EXTENSION all : behavior
 ----------
 
+`FP_CONTRACT` directive can be used to specify contraction for some floating
+point operations (more details can be found in <<floating-point-macros-and-pragmas,
+the floating-point macros and pragmas section>>).
+
+`EXTENSION` directive is used to extend the language with functionality that
+deviates from the standard behavior described in this document. Extension pragma
+directives are only needed for extensions that require changes in the language
+such that the source code is interpreted differently when the pragma is used.
+Each extension should specify whether it adds the pragma and what behavior
+will be changed due to its use in the source. If the behavior of an extension
+pragma is unspecified the implementations are not required to accept it.
+More information can be found in <<opencl-extension-spec,section 1.2 in
+the OpenCL Extension Specification>>.
```

Comment:
Minor: I can't comment on the exact line, but since these pragmas are describe briefly now the preceding paragraph should remove "whose meanings are described elsewhere" (line 3170).